browser-api/src/main/java/com/duckduckgo/browser/api/JsInjectorPlugin.kt

```diff
@@ -22,12 +22,14 @@ import com.duckduckgo.app.global.model.Site
 /** Public interface to inject JS code to a website */
 interface JsInjectorPlugin {
     /**
-     * This method is called during onPageStarted and receives a [webView] instance, the [url] of the website and the [site]
+     * This method is called during onPageStarted and receives a [webView] instance, the [url] of the website, the [site] and a boolean
+     * indicating if forced zoom is enabled in Accessibility.
      */
-    fun onPageStarted(webView: WebView, url: String?, site: Site?)
+    fun onPageStarted(webView: WebView, url: String?, site: Site?, isForcedZoomEnabled: Boolean)
```

Comment:
This is a pretty bad API that's going to end up in a nightmare. All public API changes need to go through our approval process see https://app.asana.com/0/1202561462274611/1205060262189411/f

We should be able to add that boolean to the site object though if it belongs to the site, otherwise you can probably get it directly in the implementation of this method.

Reply:
Thanks @marcosholgado ! TIL about `ContentScopeConfigPlugin` :cool: Updated the code.